### ISSUE TITLE
Timeline for remaining red squares?

### ISSUE THREAD

**Issue by nisbet-hubbard** [skarnet/s6-rc#7]
As one of those stuck with enterprise Linux, I’ve looked into the s6 constellation of software several times now as a potential drop-in replacement for (the useful parts of) systemd. And I’m really excited about the prospect of the two remaining red squares turning green as envisaged in: https://skarnet.com/projects/service-manager.html.

I was wondering if you could give us an update on the rough timeline for this last piece of the puzzle.

**Update:** in the meantime, this recent piece will be helpful to others in the same shoes https://skarnet.org/software/s6/unit-conversion.html.

**Comment by skarnet** [comment]
A comprehensive update to that document is planned for this summer. I keep saying I'll do it soon (tm) but keep getting distracted by other stuff; "this summer" should be something I manage to stick to, however.

I'm afraid not much progress has been made on s6-rc v1, at least not as much as you may hope; the main takeaway, which I will explain in the update, is that it is a *very* complex project, one or two orders of magnitude above what I am used to dealing with, and in order to feel confident enough to tackle it I needed to break down the project into several parts and acquire the necessary skills to understand how to do each part. Which I've been doing, slowly, the past year and a half.

So basically it *is* making progress, but it is slow and invisible for now, and I'm afraid I can't give you a timeline yet - or I'd risk exploding the deadlines again.

**Comment by DjogaRo** [comment]
Does the current state of https://skarnet.com/projects/service-manager.html reflect said update?
How's your tackle-confidence comming along?
Has the sponsoring allowed you to work on it full time? Have you allowed yourself to do so?
I'm not asking for a timeline. It's just, I cannot for the life of me manage to find any indicators for where it's at and I must know. :)

**Comment by skarnet** [comment]
Yeah, sorry, it got delayed again. The funding was suspended this year (for reasons that have nothing to do with me), so I did not work on it much; I hope to secure more funding for next year, but this is still uncertain.

I've had to scale back my ambitions about s6-rc v1, but I do have a path of progress, building upon s6-rc v0. I'm not talking about it much because I want to have something to show next time I go public, and unfortunately there's not much to show yet. But, at least in my head, the milestones are clearer now. I hope to be able to come back with something substantive soonish!

**Comment by DjogaRo** [comment]
Thanks. I'd say making a mental model is part of the deed, so if that got better defined, that's something.

Should you decide to do a funding campaign, please do make a mention here - now that I get mentions here relayed to my mail box. ;)
